welcome_rs/.gitignore

```diff
@@ -0,0 +1 @@
+target
```

Comment:
Please add a newline (`\n`) at the end of the file

Reply:
Also, this seems to belong to #11. Please try to keep different changes in separate PRs.